gtfs-realtime/best-practices/best-practices.md

```diff
@@ -52,7 +52,7 @@ All entities should only be removed from a GTFS Realtime feed when they are no l
 General guidelines for trip cancellations:
 
 * When canceling trips over a number of days, producers should provide TripUpdates referencing the given `trip_ids` and `start_dates` as `CANCELED` as well as an Alert with `NO_SERVICE` referencing the same `trip_ids` and `TimeRange` that can be shown to riders explaining the cancellation (e.g., detour).
-* If no stops in a trip will be visited, the trip should be `CANCELED` instead of having all `stop_time_updates` being marked as `SKIPPED`.  
+* If no stops in a trip will be visited, the trip should be `CANCELED` instead of having all `stop_time_updates` being marked as `SKIPPED`, unless the trip was a `NEW` or `DUPLICATED` trip and was subsequently canceled. 
```

Comment:
What is the usecase you have in mind with this sentence? Couldn't you just remove the trip from the feed if you don't want to see it again?

Reply:
Removing the trip from the feed is the equivalent to using schedule_relationship = `DELETED`, which is different from cancelling the trip.

A cancelled trip is still shown to the user, but removing the trip from the feed will cause it disappear, which is not what the user expects in some situations. (If the trip was added in error, it would be appropriate to remove it from the feed; if the trip was intended to run at some time and subsequently cancelled, it would still be need in the feed)

Reply:
So you want to communicate something like "earlier today we planned to add a reinforcement trip, but that has changed and it will not run"?

Reply:
Yes, that's true.